.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```

Comment:
Need to add `.idea` to the project's base `.gitignore` and remove all of these files from Git.